Sources/PointFree/Stripe.swift

```diff
@@ -8,7 +8,7 @@ import UrlFormEncoding
 public struct Stripe {
   public var cancelSubscription: (Subscription.Id) -> EitherIO<Swift.Error, Subscription>
   public var createCustomer: (Database.User, Token.Id, String?) -> EitherIO<Swift.Error, Customer>
-  public var createSubscription: (Customer.Id, Plan.Id, Int) -> EitherIO<Swift.Error, Subscription>
+  public var createSubscription: (Customer.Id, Plan.Id, Int, SubscribeData.Coupon?) -> EitherIO<Swift.Error, Subscription>
```

Comment:
can supply a coupon when subscribing now. stripe automatically handles making sure the coupon exists and that it's redeemable (can set max redemptions to 1).